pom.xml

```diff
@@ -321,6 +321,7 @@
           </execution>
         </executions>
         <configuration>
+          <source>1.8</source>
```

Comment:
Does this do anything on your machine? I still get a lot of warnings / errors. I think I will add <maven.compiler.release>8</maven.compiler.release> at the top of the pom.xml. Although that also doesn't fix the Javadoc warnings / errors when using Java 17. I you remove this line (or explain it) I will merge your PR. I had also changed the html elements locally, I used code instead of var but you know more about html then I do :)

Reply:
Ah, I didn't see that warning because I was using Java 17 which shows that warning only once and gives a lot of other warnings which Java 11 doesn't give (Java 11 is repeating the modules warning a lot of times).

I think (latest version off) Frank!Runner is using Java 17 when running Maven? You specifically used Java 11?

Can you add a comment above source 1.8 saying: Fix javadoc warning: The code being documented uses modules but the packages defined in https://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.

Reply:
Added the comment myself and fixed warning for JDK 17 and 21 too (see https://github.com/wearefrank/ladybug/commit/b79837ca529e20d8212119c4bd30d3b05e2cb5e7)